docs/user/auth-resources.md

```diff
@@ -0,0 +1,56 @@
+# Authentication resources
+
+The table below contains the routes, names, resource names, and verbs for flightctl API endpoints:
+
+|Route| Name| Resource| Verb |
+|-----|-----|---------|------|
+|`DELETE /api/v1/certificatesigningrequests`|`DeleteCertificateSigningRequests`|`certificatessigningrequests`|`deletecollection`|
+|`GET /api/v1/certificatesigningrequests`|`ListCertificateSigningRequests`|`certificatessigningrequests`|`list`|
+|`POST /api/v1/certificatesigningrequests`|`CreateCertificateSigningRequest`|`certificatessigningrequests`|`create`|
+|`DELETE /api/v1/certificatesigningrequests/{name}`|`DeleteCertificateSigningRequest`|`certificatessigningrequests`|`delete`|
+|`GET /api/v1/certificatesigningrequests/{name}`|`ReadCertificateSigningRequest`|`certificatessigningrequests`|`read`|
+|`PATCH /api/v1/certificatesigningrequests/{name}`|`PatchCertificateSigningRequest`|`certificatessigningrequests`|`patch`|
+|`PUT /api/v1/certificatesigningrequests/{name}`|`ReplaceCertificateSigningRequest`|`certificatessigningrequests`|`replace`|
+|`DELETE /api/v1/certificatesigningrequests/{name}/approval`|`DenyCertificateSigningRequest`|`certificatessigningrequests`|`deny`|
```

Comment:
Looks like we have an issue here with the CSR API. This should be PUT/PATCH on the `certificatesigningrequests/approve` subresource to either set the "approved" or "denied" conditions...